docs/README.fa.md

```diff
@@ -0,0 +1,37 @@
+<div dir="rtl">
+
+<h3 align="center"><img src="../assets/onefetch.svg" height="130px"></h3>
+
+<h5 align="center">یک ابزار کامند لاینی برای نمایش دادن اطلاعات گیت نوشته شده با زبان برنامه‌نویسی راست</h5>
+
+<p align="center">
+	<a href="https://crates.io/crates/onefetch"><img src="https://img.shields.io/crates/v/onefetch.svg" alt="cargo"></a>
+	<a href="https://github.com/o2sh/onefetch/actions"><img src="https://github.com/o2sh/onefetch/workflows/CI/badge.svg" alt="Build Status"></a>
+	<a href="https://github.com/o2sh/onefetch/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22"><img src="https://img.shields.io/github/issues/o2sh/onefetch/help%20wanted?color=green" alt="help wanted"></a>
+	<a href="./LICENSE.md"><img src="https://img.shields.io/badge/license-MIT-blue.svg"></a>
+</p>
+
+<p align="center">[<a href="https://github.com/o2sh/onefetch/docs/README.ja.md">日本語</a>, 
+<a href="https://github.com/o2sh/onefetch/docs/README.fa.md">فارسی</a>]</p>
```

Comment:
```suggestion
```
I don't think we're including links to the other READMEs in anything but the `README.md`.

For example, these aren't in `README.ja.md`.